tests/backup/backup_helper.go

```diff
@@ -9,6 +9,7 @@ import (
 	"os/exec"
 	"reflect"
 	"regexp"
+	"sigs.k8s.io/kustomize/kyaml/sliceutil"
```

Comment:
Why is this package needed?

Reply:
It's used in line 5236 for checking if a namespace is in a list of namespaces or not

Reply:
We already have a function like this in `backup_helper.go`

```
// IsPresent verifies if the given data is present in slice of data
func IsPresent(dataSlice interface{}, data interface{}) bool {
	s := reflect.ValueOf(dataSlice)
	for i := 0; i < s.Len(); i++ {
		if s.Index(i).Interface() == data {
			return true
		}
	}
	return false
}
```

Reply:
What's the benefit of using this over the one which I have used?

Reply:
I do not see any difference between these two functions and it will avoid an import.

Reply:
Done, I still don't understand why we need it.